test/test.c

```diff
@@ -24333,6 +24365,35 @@ test_JetStreamMgtConsumers(void)
     jsConsumerInfo_Destroy(ci);
     ci = NULL;
 
+    if (serverVersionAtLeast(2, 11, 0))
+    {
+        test("Pause consumer: ");
+        jsConsumerPauseResponse *cpr = NULL;
+        s = js_PauseConsumer(&cpr, js, "MY_STREAM", "my_name", TIME_20350101, NULL, &jerr);
+        testCond((s == NATS_OK) && (jerr == 0) && (cpr != NULL)
+            && cpr->Paused
+            && (cpr->PauseUntil == TIME_20350101)
+            && (cpr->PauseRemaining > 0));
+        NATS_FREE(cpr);
```

Comment:
Would use a `jsConsumerPauseResponse_Destroy` here.. will be more portable if one day some other field is present that would need to be freed.